src/engine.js

```diff
@@ -98,36 +63,71 @@ const editTaskDescription = (index, newDescription) => {
     saveTasksToLocalStorage();
     displayItemsByIndex();
   } else {
-    alert('Invalid task index.');
+    alert('please add a task to init app');
   }
 };
 
-// Event listeners for the checkboxes and delete buttons
-const attachEventListeners = () => {
+// Display tasks
+const displayItemsByIndex = () => {
+  listContainer.innerHTML = '';
+
   list.forEach((item) => {
-    const { index } = item;
-    const completedRadio = document.querySelector(`input[name="completed_${index}"]`);
+    const { description, completed, index } = item;
+
+    const listItem = document.createElement('div');
+    const descriptionSpan = document.createElement('span');
+    const completedRadio = document.createElement('input');
+    const edit = document.createElement('button');
+    const hr = document.createElement('hr');
+
+    edit.classList.add('edit-task');
+
+    listItem.classList.add('icon');
 
+    descriptionSpan.textContent = description;
+    completedRadio.type = 'checkbox';
+    completedRadio.name = `completed_${index}`;
+    completedRadio.checked = completed;
+    edit.textContent = 'Edit task';
```

Comment:
- [ ] As we can see 👀 in the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) there isn't an `Edit task` message near the task. Editing tasks is available when the user double clicks on the tasks, it paints in yellow and can write a new task over, and it actualizes in the local storage. Please 🙏🏽 remove the `Edit task` message link and correct the behavior to be similar to the wireframe.